packages/modules/viewer-react/src/components/BaseViewer.scss

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@bentley/ui-core/lib/ui-core/style/themecolors.scss";
+
+.not-signed-in-container {
```

Comment:
Might be better to use flexbox than absolutely positioning this. We never know what the size of the viewer's container is going to be.